cmd/di.go

```diff
@@ -173,6 +173,7 @@ func (di *Dependencies) Bootstrap(nodeOptions node.Options) error {
 	nats_discovery.Bootstrap()
 
 	log.Infof("Starting Mysterium Node (%s)", metadata.VersionAsString())
+	log.Infof("Build information (%s)", metadata.BuildAsString())
```

Comment:
Maybe remake `VersionAsString()` function, so that that we would have build info in all places

Reply:
@Waldz This would require a lot of changes like metrics, tequilapi, etc. If that's needed lets do it in another PR. I need this as this easily solves the current issue at hand.